ui/lobby/css/_spotlight.scss

```diff
@@ -7,6 +7,65 @@
 .tour-spotlight {
   @extend %flex-center-nowrap, %nowrap-hidden, %page-metal;
 
+  &.highlighted {
+    @extend %active;
+
+    .selected-color-original & {
+      background: $c-color-choice-blue;
+      color: $c-color-choice-font-default;
```

Comment:
I'll have to factorize this line since it's now common for all highlighted events in the spotlight